test/exprtest/CMakeLists.txt

```diff
@@ -28,9 +28,15 @@ set(SYCLBLAS_EXPRTEST ${CMAKE_CURRENT_SOURCE_DIR})
 set(SYCL_EXPRTEST_SRCS
   ${SYCLBLAS_EXPRTEST}/blas1_scal_asum_test.cpp
   ${SYCLBLAS_EXPRTEST}/blas1_axpy_copy_test.cpp
-  ${SYCLBLAS_EXPRTEST}/collapse_nested_tuple.cpp
 )
 
+# Temporary disabling the following tests for Intel DPC++ as currently Intel compiler crashes while running the following tests
+if(is_computecpp)
+  set(SYCL_EXPRTEST_SRCS
+    ${SYCLBLAS_EXPRTEST}/collapse_nested_tuple.cpp
```

Comment:
I think you meant to append this test to the list in that case, didn't you?

Reply:
Yes. Fixed this.